sqlglot/dialects/hive.py

```diff
@@ -285,9 +285,7 @@ class Parser(parser.Parser):
         }
 
         def _parse_transform(self) -> exp.Transform | exp.QueryTransform:
-            args = self._parse_csv(self._parse_lambda)
-            self._match_r_paren()
-
+            args = self._parse_wrapped_csv(self._parse_lambda)
```

Comment:
don't you need to consume a l paren here?

Reply:
that's handled by the `_parse_wrapped_csv` method right?

Reply:
ah ok